#goal definition
string named_pose
---
#result definition
---
#feedback
